src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -11,13 +11,33 @@ public function __construct($key, $usergroup)
     }
 }
 
+class PropertyKeyNotAllowedException extends \RuntimeException
+{
+    public function __construct($key)
+    {
+        parent::__construct(sprintf('Property key "%s" is not allowed.', $key));
+    }
+}
+
 class Property
 {
+    const RESERVED_PROPERTY_KEYS = [
```

Comment:
Please use a doc block do explain what the key patterns are reserved for:

* `/image\d+/`: Image URLs of default type.
* `/thumbnail\d+/`: Image URLs of type thumbnail.
* `ordernumber`: The products first exported ordernumber.

Reply:
Done